daemons/controld/controld_schedulerd.c

```diff
@@ -470,8 +479,8 @@ do_pe_invoke_callback(xmlNode * msg, int call_id, int rc, xmlNode * output, void
         crm_debug("Re-asking for the CIB: %d other peer updates still pending",
                   (num_cib_op_callbacks() - 1));
         sleep(1);
```

Comment:
Remember to get rid of the sleep call - that's the whole point of moving the following code into a mainloop timer.